python/ray/train/examples/accelerate/accelerate_torch_trainer_no_raydata.py

```diff
@@ -20,6 +20,7 @@
     set_seed,
 )
 
+import ray.data
```

Comment:
Do we need this here? The file name implies "no ray data"

Reply:
Oh you are right! let me delete this!